polyfill/lib/ecmascript.mjs

```diff
@@ -1140,6 +1161,17 @@ export const ES = ObjectAssign({}, ES2020, {
     const cal2 = ES.CalendarToString(two);
     return cal1 === cal2;
   },
+  ConsolidateCalendars: (one, two) => {
+    const sOne = ES.CalendarToString(one);
+    const sTwo = ES.CalendarToString(two);
+    if (sOne === sTwo || sOne === 'iso8601') {
```

Comment:
I thought we had removed the special handling of the ISO calendar as a "null" calendar everywhere else ­— double check with @sffc who probably remembers better than I do.

Reply:
I'll reach out to @sffc about this, thanks.